### PR TITLE
Homework 2: HTML 02, HTML 03, HTML 04

### PR DESCRIPTION
**Pull request by oldarol**
Could you please check the 2nd homework?
Thanks,
S.Busel

### CODE REVIEW

**Review by pluhin — COMMENTED** [review]
In python files could you please align all markup for all code sections

File: sbusel/HTML_01/Task_1_3_Additional.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <p align="center"><strong>Магазин работает:</strong></p>
+        <p align="center">c 08<sup><ins>30</ins></sup> до 18<sup><ins>30</ins></sup><br>
+        обед с 13<sup><ins>00</ins></sup> до 14<sup><ins>00</ins></sup><br>
+        выходной: <ins>суббота</ins>, <ins>воскресенье</ins></p>
+        <hr>
+        <h3>Вот такой текст</h3>
+        <h2><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h2>
+        <hr>
+        <h3>Акроним и курсив</h3>
+        <p>ТФ: Y(<em>x</em>)=e­<sup><em>-iπx</em></sup>+log<sub><em>x</em></sub>4</p>­
```
Comment (pluhin) [inline sbusel/HTML_01/Task_1_3_Additional.html]: Need replace pi -> mnemonic

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — COMMENTED** [review]
Could you please align all code section to one markup format

File: sbusel/HTML_01/Task_1_3_Additional.html (hunk 1 above)
Comment (oldarol) [reply sbusel/HTML_01/Task_1_3_Additional.html]: Got it.

**Pull request merged**